dbt_project.yml

```diff
@@ -20,7 +20,10 @@ models:
 
 seeds:
   synthea_omop_etl:
+    vocabulary:
+      +enabled: true
```

Comment:
i want to figure out how to disable these dynamically if someone is doing BYO data mode. unfortunately this is not possible using `vars` - https://github.com/dbt-labs/dbt-core/issues/4873